hyperdome/client/settings_dialog.py

```diff
@@ -716,11 +710,7 @@ def data_dir_button_clicked(self):
         )
 
         if selected_dir:
-            self.common.log(
-                "SettingsDialog",
-                "data_dir_button_clicked",
-                "selected dir: {}".format(selected_dir),
-            )
+            self.__log.info(f"selected dir: {selected_dir}")
```

Comment:
f"{selected_dir=}" is cooler tho

Reply:
Dang I even did that for a lot of other old logging statements, oh well